primap2/tests/data/test_read_wide_csv_file_output_unit_def.csv

```diff
@@ -1,4 +1,4 @@
 ,source,scenario (general),area (ISO3),entity,unit,category (IPCC2006),Class (class),Type (type),1991,2000,2010
-0,TESTcsv2021,HISTORY,AUS,CO2,Gg CO2 / yr,1,TOTAL,fugitive,4,5,6
-1,TESTcsv2021,HISTORY,ZAM,CH4,Gg CH4 / yr,2,TOTAL,fugitive,7,8,9
-2,TESTcsv2021,HISTORY,ZAM,CO2,Gg CO2 / yr,2,TOTAL,fugitive,12,13,14
+0,TESTcsv2021,HISTORY,AUS,CO2,Gg CO2 / yr,1,TOTAL,fugitive,4.0,5.0,6.0
```

Comment:
We did get problems mixing data with and without decimal places, which is why I added `.astype(float)` for data columns in the reading functions and added some decimal places randomly so that this "mixed" data is properly tested.